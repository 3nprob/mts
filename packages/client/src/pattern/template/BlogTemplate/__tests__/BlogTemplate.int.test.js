import React from 'react'
import { defaultRender } from './BlogTemplate.int.render'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<BlogTemplate />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
