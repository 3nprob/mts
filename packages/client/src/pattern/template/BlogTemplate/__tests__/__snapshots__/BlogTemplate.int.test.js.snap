// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogTemplate /> Snaphot must match defaultRender 1`] = `
<div
  className="blogTemplate"
>
  <ul
    className="buttonList"
  >
    <li
      className="buttonListItem"
    >
      <a
        aria-label="Share On Social Media: facebook"
        href="/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          height="30px"
          viewBox="0 0 96.124 96.123"
          width="30px"
        >
          <path
            d="M72.089.02L59.624 0C45.62 0 36.57 9.285 36.57 23.656v10.907H24.037a1.96 1.96 0 0 0-1.96 1.961v15.803a1.96 1.96 0 0 0 1.96 1.96H36.57v39.876a1.96 1.96 0 0 0 1.96 1.96h16.352a1.96 1.96 0 0 0 1.96-1.96V54.287h14.654a1.96 1.96 0 0 0 1.96-1.96l.006-15.803a1.963 1.963 0 0 0-1.961-1.961H56.842v-9.246c0-4.444 1.059-6.7 6.848-6.7l8.397-.003a1.96 1.96 0 0 0 1.959-1.96V1.98A1.96 1.96 0 0 0 72.089.02z"
          />
        </svg>
      </a>
    </li>
    <li
      className="buttonListItem"
    >
      <a
        aria-label="Share On Social Media: twitter"
        href="/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          height="30px"
          viewBox="0 0 310 310"
          width="30px"
        >
          <path
            d="M302.973 57.388a117.512 117.512 0 0 1-14.993 5.463 66.276 66.276 0 0 0 13.494-23.73 5 5 0 0 0-7.313-5.824 117.994 117.994 0 0 1-34.878 13.783c-12.381-12.098-29.197-18.983-46.581-18.983-36.695 0-66.549 29.853-66.549 66.547 0 2.89.183 5.764.545 8.598C101.163 99.244 58.83 76.863 29.76 41.204a5.001 5.001 0 0 0-8.196.642c-5.896 10.117-9.013 21.688-9.013 33.461 0 16.035 5.725 31.249 15.838 43.137a56.37 56.37 0 0 1-8.907-3.977 5 5 0 0 0-7.427 4.257c-.007.295-.007.59-.007.889 0 23.935 12.882 45.484 32.577 57.229a57.372 57.372 0 0 1-5.063-.735 4.998 4.998 0 0 0-5.699 6.437c7.29 22.76 26.059 39.501 48.749 44.605-18.819 11.787-40.34 17.961-62.932 17.961a120.4 120.4 0 0 1-14.095-.826 5 5 0 0 0-3.286 9.174c29.023 18.609 62.582 28.445 97.047 28.445 67.754 0 110.139-31.95 133.764-58.753 29.46-33.421 46.356-77.658 46.356-121.367 0-1.826-.028-3.67-.084-5.508 11.623-8.757 21.63-19.355 29.773-31.536a5 5 0 0 0-6.182-7.351z"
          />
        </svg>
      </a>
    </li>
    <li
      className="buttonListItem"
    >
      <a
        aria-label="Share On Social Media: linkedin"
        href="/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          height="30px"
          viewBox="0 0 310 310"
          width="30px"
        >
          <path
            d="M72.16 99.73H9.927a5 5 0 0 0-5 5v199.928a5 5 0 0 0 5 5H72.16a5 5 0 0 0 5-5V104.73a5 5 0 0 0-5-5zM41.066.341C18.422.341 0 18.743 0 41.362 0 63.991 18.422 82.4 41.066 82.4c22.626 0 41.033-18.41 41.033-41.038C82.1 18.743 63.692.341 41.066.341zM230.454 94.761c-24.995 0-43.472 10.745-54.679 22.954V104.73a5 5 0 0 0-5-5h-59.599a5 5 0 0 0-5 5v199.928a5 5 0 0 0 5 5h62.097a5 5 0 0 0 5-5V205.74c0-33.333 9.054-46.319 32.29-46.319 25.306 0 27.317 20.818 27.317 48.034v97.204a5 5 0 0 0 5 5H305a5 5 0 0 0 5-5V194.995c0-49.565-9.451-100.234-79.546-100.234z"
          />
        </svg>
      </a>
    </li>
  </ul>
  <nav
    className="nav"
  >
    <div
      className="navBar"
    >
      <a
        className="navBarLogo"
        href="/"
      >
        <img
          alt="logo"
          height="40"
          src={Object {}}
        />
      </a>
      <ul
        className="navBarList"
      >
        <li
          className="navBarListItem"
        >
          <a
            aria-label="About Page"
            className="navBarListItemLink"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          className="navBarListItem"
        >
          <a
            aria-label="Blog Page"
            className="navBarListItemLink"
            href="/blog"
          >
            Blog
          </a>
        </li>
        <li
          className="navBarListItem"
        >
          <a
            aria-label="Courses Page"
            className="navBarListItemLink"
            href="/courses"
          >
            Courses
          </a>
        </li>
      </ul>
      <div
        className="navBarMobileToggle"
      >
        <button
          className="navBarMobileToggleButton"
          onClick={[Function]}
          type="button"
        >
          <svg
            height="24px"
            viewBox="0 0 124 124"
            width="24px"
          >
            <path
              d="M112 6H12C5.4 6 0 11.4 0 18s5.4 12 12 12h100c6.6 0 12-5.4 12-12s-5.4-12-12-12zM112 50H12C5.4 50 0 55.4 0 62s5.4 12 12 12h100c6.6 0 12-5.4 12-12s-5.4-12-12-12zM112 94H12c-6.6 0-12 5.4-12 12s5.4 12 12 12h100c6.6 0 12-5.4 12-12s-5.4-12-12-12z"
            />
          </svg>
        </button>
      </div>
    </div>
  </nav>
  <div
    className="topBorder"
  >
    <label
      className="label"
      htmlFor="newsletter"
    >
      <div
        className="labelSectionHeader"
      >
        <h2
          className="labelSectionHeaderH2"
        >
          Take Part
        </h2>
        <h3
          className="labelSectionHeaderH3"
        >
          Never miss an article about web development and JavaScript.
        </h3>
      </div>
      <p
        className="labelLead"
      >
        <svg
          height="20px"
          viewBox="0 0 123.811 123.811"
          width="20px"
        >
          <path
            d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
          />
        </svg>
        Join 0+ Developers
      </p>
      <p
        className="labelLead"
      >
        <svg
          height="20px"
          viewBox="0 0 123.811 123.811"
          width="20px"
        >
          <path
            d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
          />
        </svg>
        Learn Web Development with JavaScript
      </p>
      <p
        className="labelLead"
      >
        <svg
          height="20px"
          viewBox="0 0 123.811 123.811"
          width="20px"
        >
          <path
            d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
          />
        </svg>
        Tips and Tricks
      </p>
      <p
        className="labelLead"
      >
        <svg
          height="20px"
          viewBox="0 0 123.811 123.811"
          width="20px"
        >
          <path
            d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
          />
        </svg>
        Access Tutorials, eBooks and Courses
      </p>
      <p
        className="labelLead"
      >
        <svg
          height="20px"
          viewBox="0 0 123.811 123.811"
          width="20px"
        >
          <path
            d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
          />
        </svg>
        Personal Development as a Software Engineer
      </p>
      <div
        style={
          Object {
            "marginBottom": "20px",
          }
        }
      />
      <form
        className="labelForm"
      >
        <input
          autoComplete="email"
          className="labelFormInput"
          id="newsletter"
          placeholder="Your email address"
          type="email"
        />
        <button
          className="labelFormButton"
          type="submit"
        >
          Subscribe
        </button>
      </form>
      <small>
        <small>
          View ourÂ 
          <a
            href="/legal"
            itemProp="url"
          >
            Privacy Policy
          </a>
          .
        </small>
      </small>
    </label>
  </div>
</div>
`;
